frontend/src/Editor/Viewer/Confirm.jsx

```diff
@@ -69,10 +69,10 @@ export function Confirm({
       )}
       <Modal.Body data-cy={'modal-message'}>{message}</Modal.Body>
       <Modal.Footer className="mt-3">
-        <button className="btn" onClick={handleClose} data-cy={'modal-cancel-button'}>
+        <button className="btn btn-danger" onClick={handleClose} data-cy={'modal-cancel-button'}>
           {cancelButtonText === '' ? t('globals.cancel', 'Cancel') : cancelButtonText}
         </button>
-        <button className="btn btn-danger" onClick={handleConfirm} data-cy={'modal-confirm-button'}>
+        <button className="btn" onClick={handleConfirm} data-cy={'modal-confirm-button'}>
```

Comment:
Changing this will affect through out the `Confirm` component wherever it is using. Attaching the screenshot - Delete popup also got affected

<img width="402" alt="image" src="https://user-images.githubusercontent.com/50441969/236786574-35a6ba0c-4ca2-463f-aa7c-8547c3c6015b.png">